src/account/UpgradeableModularAccount.sol

```diff
@@ -21,12 +21,13 @@ import {_coalescePreValidation, _coalesceValidation} from "../helpers/Validation
 
 import {DIRECT_CALL_VALIDATION_ENTITYID, RESERVED_VALIDATION_DATA_INDEX} from "../helpers/Constants.sol";
 
-import {ExecutionManifest} from "../interfaces/IExecution.sol";
-import {IExecutionHook} from "../interfaces/IExecutionHook.sol";
+import {IExecutionHookModule} from "../interfaces/IExecutionHookModule.sol";
```

Comment:
Oof, this is really wordy...

Reply:
I actually think it's fine?

Alternatively, I'd be okay with omitting the Module suffix, but changing `IValidation` and `IExecution` to `IValidator` and `IExecutor`, respectively. This does unfortunately overlap with `IStandardExecutor` though...

Reply:
On second thought, this is OK with me.